R/layouts.R

```diff
@@ -251,3 +251,27 @@ split_layout <- function(..., cell_widths = NULL, cell_args = "", style = NULL){
 splitLayout <- function(..., cellWidths = NULL, cellArgs = "", style = NULL) {
   split_layout(..., cell_widths = cellWidths, cell_args = cellArgs, style = style)
 }
+
+#' @export
+#' @rdname flow_layout
+flow_layout <- function(..., cell_args = list(), item_width = "208px", column_gap = "12px", row_gap = "0px") {
+  container_style <- glue::glue(
+    "display: grid;",
+    "grid-template-columns: repeat(auto-fill, {item_width});",
+    "column-gap: {column_gap};",
+    "row-gap: {row_gap};"
+  )
+  item_style <- "align-self: start;"
+  args <- splitArgs(...)
+  children <- lapply(args$positional, function(child) {
+    do.call(shiny::tags$div, c(style = item_style, cellArgs, list(child)))
```

Comment:
I think `cellArgs` should have default value here:
`do.call(shiny::tags$div, c(style = item_style, cellArgs = "", list(child)))`
If not that example from Dominik's comment doesn't work

Reply:
It didn't work before as I forgot to rename `cellArgs` to `cell_args`. The default value should be in the function definition, otherwise user cannot overwrite it.